examples/maintained/cgi-bin/dataflow.yaml

```diff
@@ -223,15 +223,15 @@ operations:
     conditions:
     - no_git_branch_given
     inputs:
-      repo: git_repository
+      record: git_repository
     name: git_repo_default_branch
     outputs:
       branch: git_branch
     stage: processing
   git_repo_release:
     inputs:
       branch: git_branch
-      repo: git_repository
+      record: git_repository
```

Comment:
Everything in this file (`examples/maintained/cgi-bin/dataflow.yaml` should be repo ( so no need to change anything in this file )